server/models/idea.js

```diff
@@ -0,0 +1,45 @@
+const MongoClient = require('mongodb').MongoClient;
+const url = 'mongodb://localhost:27017/rankerdb';
+
+exports.all = function(callback) {
+	MongoClient.connect(url, function(err, db) {
+	  if(err)throw(err);
+
+  	db.collection('ideas').find().toArray(function(err, result) {
+  		if (err) throw err;
+  		callback(result);
+  	});
+	});
+};
+
+exports.get = function(name, callback) {
+	MongoClient.connect(url, function(err, db) {
+	  if(err)throw(err);
+
+	  db.collection('ideas').find({name: name}).toArray(function(err, result) {
```

Comment:
Probably isn't the best idea to query by the name of the idea, since different people might create ideas with the same name. It's probably better to use the _id field since that's guaranteed to be unique.